### ISSUE TITLE
[Question] Why getNearestDc doesn't use?

### ISSUE THREAD

**Issue by Choochmeque** [Aseman-Land/libqtelegram-aseman-edition#10]
I think, it's affect https://github.com/Aseman-Land/Cutegram/issues/104

**Comment by realbardia** [comment]
Yes... Maybe It can fix this problem. I'm working on telegram api 29 and many other changes on libqtelegram-ae. I'll also add this feature soon :)

**Comment by no-identd** [comment]
See also https://github.com/Aseman-Land/Cutegram/issues/65#issuecomment-109931552
